bot/cogs/common.py

```diff
@@ -38,8 +41,7 @@ async def ping(self, ctx: Context) -> None:
     # TODO : after db integration, add Time Limit, and grand announcement, when the poll is over.
     @command(aliases=("poll",))
     async def vote(self, ctx: Context, title: str, *options: str) -> None:
-        """
-        Build a quick voting poll with matching reactions with the provided options.
+        """Build a quick voting poll with matching reactions with the provided options.
```

Comment:
It must be on the first line though. Furthermore, that's how `Rapptz` himself writes it. [I'm sure we can trust him](https://github.com/Rapptz/discord.py/blob/b77af199392209017e59eff366b46354fa210f64/discord/emoji.py#L33)

Reply:
`Rapptz` uses a very specific docstring formatting while other popular projects use a different one, this is really a matter of preference, each repository can follow some more common or its own docstring directive, I've already documented ours in `CONTRIBUTING.md` you can read through it but the above suggestion already shows the docstring format which we follow so respect that and adapt all of the docstrings to respect this directive.

Reply:
It's not "very specific" : even my linter agrees with him

Reply:
It is, most liners will agree with his docstring format as well as the `google` docstring format or the `NumPy` docstring format or `Sphinx` or any other of the docstring formatting convention, but our convention is not so different and most linters will agree with it too, if yours doesn't I'd suggest changing a linter specifically for our project to what we enforce using anyway `flake8` with our config from `tox.ini` that will save you a lot of issues regarding the differences of these linters or once again adding this specific error to ignored formatting errors/warning for our project

Reply:
Actually, to configure my linter I'd need an extra `setup.cfg` file... That's why I don't do it.
And why don't you follow every linter on earth ?

Reply:
It doesn't support `tox.ini`

Reply:
Because following every linter on earth would be literally impossible, linters often disagree with each other as to what's the correct syntax. 

We decided to follow `flake8` and all contributors will need to respect that even if it's not their preferred linter. We use `tox.ini` for the configuration of `flake8` you should be able to configure your linter to respect that but even if you don't do that, I'd think your IDE/Text Editor provides you with an option to have per-project settings, for example in vscode this would be `.vscode/settings.json` which override the global settings, in there you should be able to make a change to how your linter works by ignoring certain errors, in vscode, this would be:
```json
{
    "python.linting.flake8Args": [
        "--max-line-length=150",
        "--ignore=ANN101,ANN102,ANN001,ANN002,ANN003,E731"
    ]
}

Reply:
I'm not using flake8 : I'm using an Atom package providing 4 linting tools, with all settings written in a single `setup.cfg` file
And my meaning was, if nearly every single linter agree on this point, why do you disagree ?

Reply:
As I described before I disagree because of the readability loss and no real benefit of following this except that your linters will pass and since we don't use those liners in this repository we don't need to comply with them.

You should be able to configure Atom to use `flake8` as is required in our repository, if not, disable the linting entirely and check your code with `pipenv run lint` since those are the standards that we require, you can also keep the linting on and configure it in a way where you'll ignore all errors except those which we follow.

I get that your linters doesn't agree with some of our conventions, but if we had thousands of contributors and each of them with different sets of linters which we should all respect since those linters goes against our standards we'd end up in chaos where we'd need to change the repository syntax rules constantly and rewriting the code and pre-commit checks to comply with them. 

We simply follow `flake8` with the addition of `flake8-bugbear`, nothing else and we've created our guidelines based on that, every contributor is required to respect these.

Reply:
1) your tox.ini file doesn't contain anything, except errors that I am not getting, and the max_len, but I'm already ignoring those errors, or many lines would've already been cut to less than 79. Therefore, changing my linter is useless.
2) Why do you say that it's "less readable" ? It's just as much readable. And no, I'm not changing it rn, Imma sleep

Reply:
1. I'm not saying to change your liner since it doesn't respect the correct max line length, I'm saying it since it's different from what we use (`flake8`) which makes it incompatible with our repository unless you want to ignore many of the linting errors you're getting, I'm not sure what's wrong with it I'm not asking you to change your linters everywhere, I'm asking you to respect ours you don't need to completely remove Atoms linting, just to override it on our repository, it should be configurable to set a different linter for our repository and use the default for your things. You can't expect us to respect all linting errors from liners which we do not use and I'm not even forcing you to do this, as I said if you don't want to you can ignore all the errors from your linters by configuring that in per-project settings if that's impossible with Atom, either change editors or change your whole linting or make that `setup.cfg` or ignore those errors and don't enforce them here.

2. Readability is a matter of opinion, but I find it easier to read
```py
"""
Some title here

Some description
"""
```
in comparison to
```py
"""Some title here

Some description
"""
```
Since part of a docstring definition (`"""`) doesn't get into your way
I'm not saying it improves readability by a great deal, but my opinion is that it's better and it is what our repository follows.